src/clj/chronograph/domain/user.clj

```diff
@@ -2,6 +2,8 @@
   (:require [chronograph.db.core :as db]
             [next.jdbc :as jdbc]
             [chronograph.db.google-profile :as google-profile-db]
+            [chronograph.db.acl :as acl-db]
```

Comment:
nit: this ns is required as `acl-db` here, and `db-acl` elsewhere. Prefer one of those for consistency, and autocomplete goodness. We've used the `db-foo` convention thus far.